### DISCUSSION TITLE
Langserve Quickstart Example Not found error

### DISCUSSION THREAD

**Discussion by Pavan-628** [langchain-ai/langserve#619] [Q&A]
I am following the langserve quickstart example. here's the link

https://python.langchain.com/docs/get_started/quickstart/#serving-with-langserve

However this example fails with detail: Not found error when launching the agent with python serve.py

Some body pls help

**Comment by javaeeeee** [comment]
The `curl` request looks like:

```bash
curl http://localhost:8000/chain/invoke -d '{"input":{"language": "Italian", "text": "hi"}}' -H 'Content-Type: application/json'
```

First, you need to add `/invoke`, '/stream' to the URL and second, the body of the request contains `input` field. 

The explanation can be found in a blog post https://blog.langchain.dev/introducing-langserve/.
